storage/remote/queue_manager_test.go

```diff
@@ -711,22 +711,26 @@ func TestShouldReshard(t *testing.T) {
 		startingShards                           int
 		samplesIn, samplesOut, lastSendTimestamp int64
 		expectedToReshard                        bool
+		sendDeadline                             model.Duration
```

Comment:
Where is this used?

Reply:
Don't remember exactly what happend, adding this to the config below this either got missed or broken in a rebase/cherry-pick from kushals commit. Resolved further below in the creation of the `QueueManager` in the loop further below.